html/elements/blink.json

```diff
@@ -0,0 +1,61 @@
+{
+  "html": {
+    "elements": {
+      "blink": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/blink",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "1",
+              "version_removed": "22"
+            },
+            "firefox_android": {
+              "version_added": false,
```

Comment:
This should be "1" ? Same as Firefox desktop?

Reply:
Yes, fixed.